_1327/documents/tests.py

```diff
@@ -707,6 +707,7 @@ def setUpTestData(cls):
 		cls.document.set_all_permissions(baker.make(Group))
 
 	def test_render_text_no_permission(self):
+		print("render test no permission")
```

Comment:
Only the prints need to be removed, and then we're ready to go!

Reply:
done!